### ISSUE TITLE
The conflict is caused by:     fairseq 0.12.2 depends on omegaconf<2.1     hydra-core 1.0.7 depends on omegaconf<2.1 and >=2.0.5

### ISSUE THREAD

**Issue by pythonwhizghh** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#2244]
INFO: pip is looking at multiple versions of hydra-core to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install -r C:\Users\(User name)\Downloads\Retrieval-based-Voice-Conversion-WebUI-main\Retrieval-based-Voice-Conversion-WebUI-main\requirements-dml.txt (line 7) and fairseq because these package versions have conflicting dependencies.

The conflict is caused by:
    fairseq 0.12.2 depends on omegaconf<2.1
    hydra-core 1.0.7 depends on omegaconf<2.1 and >=2.0.5
I'm running on Intel GPU, windows, python 3.9

**Comment by crack-dawn** [comment]
Perhaps using Docker environment is a better choice for you.

**Comment by heartgold512** [comment]
"python 3.9" thats your problem the github repo doesnt work with that as you will get collisions.
 It appears python 3.10.0 works so maybe try earlier version of python

**Comment by heartgold512** [comment]
yeah creating a venv (virtual) of python version 3.10.0 worked for me

**Comment by sebdufourcq** [comment]
You have to downgrade to pip 24.0
`python.exe -m pip install pip==24.0`

**Comment by werbals** [comment]
Tried python 3.10 with no luck. pip 24.0 also didn't help.

**Comment by KirGear** [comment]
> Tried python 3.10 with no luck. pip 24.0 also didn't help.

@werbals try to use python 3.10.0rc1 
You can also show me the error logs and maybe I can tell you something else.

**Comment by german-leontiev** [comment]
> You have to downgrade to pip 24.0 `python.exe -m pip install pip==24.0`

Helped with new conda 3.9 env
Also I've noticed this warning.

```
DEPRECATION: omegaconf 2.0.6 has a non-standard dependency specifier PyYAML>=5.1.*. pip 24.1 will enforce this behaviour change. A possible replacement is to upgrade to a newer version of omegaconf or contact the author to suggest that they release a version with a conforming dependency specifiers. Discussion can be found at https://github.com/pypa/pip/issues/12063
```

So, maybe someone who knows authors will tell them?

**Comment by Aksolot146** [comment]
> You have to downgrade to pip 24.0 `python.exe -m pip install pip==24.0`

how to get rid of auto-update pip while installing RVC?

**Comment by SanGuang-Li** [comment]
first you need to  use python 3.10   ,as the same time  ,you  need  python.exe -m pip install pip==24.0, I have   advice  you'd  better  user anaconda to complete this pip.

**Comment by yoruniubi** [comment]
> You have to downgrade to pip 24.0 `python.exe -m pip install pip==24.0`

It worked for me

**Comment by lynnalis** [comment]
> first you need to use python 3.10 ,as the same time ,you need python.exe -m pip install pip==24.0, I have advice you'd better user anaconda to complete this pip.

Worked for me! Thank you! :)

**Comment by Ruinique** [comment]
> You have to downgrade to pip 24.0 `python.exe -m pip install pip==24.0`

Yes, it works. Thx!

**Comment by Rajeev0651** [comment]
> You have to downgrade to pip 24.0 `python.exe -m pip install pip==24.0`

worked

**Comment by RVC-Boss** [comment]
You can update the code to the latest `main` branch, where this issue has already been resolved. The latest `main` branch has resolved the dependency issues.
